docs/content/docs/scale.md

```diff
@@ -0,0 +1,110 @@
+### Memory
+- Memory per pod/envoy onboarded in the network is calculated after the initial snapshot with nothing onboarded on the mesh is seen to take into account standalone memory used by OSM.
+  - Initial memory use of OSM **40~46MB**
+  - Memory increase per pod ranges between **800KB~1MB** per pod. 
+- With **256MB** max memory limit for OSM's container, **180-200 pods** should be a conservative limit. Any more pods
+can (and will) have OSM `OOMKilled` by Kubernetes.
+- With **512MB** max memory limit for OSM's container, it can reach to **~400 pods**, although convergence times will spike through the roof up to **+120s** with a rather oversubscribed CPU profile, that could potentially lead to other issues.
+
+### CPU
+#### 1.5/2 vCPU (512MB mem)
+- At **200 pods**, ADS completion times<sup>[1]</sup> for individual envoys range around **10s-20s** and full configuration convergence ranges around **20-40s**
+- At **400 pods**, ADS completion times<sup>[1]</sup> for individual envoys range around **40-90s** and full configuration convergence ranges around **+150s**
+- Injector webhook and certificate (2048 RSA) issuance times are stable between **1s-5s** in all cases.
+#### 4 vCPU (512MB mem)
+- At **200 pods**, ADS completion times<sup>[1]</sup> for individual envoys range around **5-10s**, however full configuration convergence falls into the **20-40s** bucket.
+- At **400 pods**, ADS completion times<sup>[1]</sup> for individual envoys range around **20-40s** and full configuration convergence ranges around **+90s**
+- Injector webhook and certificate (2048 RSA) issuance times are stable between **1s-5s** in all cases.
+
+- In both cases, and taking into account the architecture limitations explained above, OSM will run hot on CPU, most of the time recomputing ADS updates for all pods in the mesh. This oversubscription could lead to not enough cycles available for other critical path routines (liveness), depending entirely on the scheduler.
+- Adding more vCPU helps OSM compute a larger amount of ADS updates at a time, as well as reduce the configuratin convergence time - however due to the nature of the buckets, we suspect this is far from a linear improvement as some of these numbers could suggest.
+
+
+Below, more results with meaningful related data explained.
+
+## Detailed Test Results
+#### 256 RAM @1.5 CPU
+- OSM is killed (OOM) at around ~260 Pods
+- mem/pod on OSM is grossly around 787kB
+- Full CPU utilization is first seen at around ~180-200 pods.
+- At around 200 pods, ADS updates get bucketed at between 10-20s to 20-40s seconds collectively<sup>[1]</sup>. 
+- Certificate issuance: constant, going from 1s to 5s buckets at most, with some non-trended outliers.
+- Webhook injector: constant, ranging also from 1s to 5s, with some non-trended outliers.
+
+[CPU profile during test](/docs/content/docs/images/scale/256-1.5-cpu.png)
```

Comment:
You do a great job of recapping the main milestones in terms of number of pods in the bulleted list above. If there was a way to capture that directly in these graphs, that would be great to have in addition.

Reply:
I have a lot more data but I couldn't fit it all here for the runs, plus it looses the interactivity of Grafana, which is a big plus to explore the data.
I added mostly that show at least correlation with some of the statements or analysis provided on the doc.
I can (of course) add more if needed (iteration times, envoys onboarded per time, cpu avgs for iteration, etc.)